src/app/entities/auth/components/login-form/login-form.component.html

```diff
@@ -1,4 +1,4 @@
-<form class="login-form card" (submit)="onFormSubmit()">
+<form class="login-form card" #form="ngForm" (submit)="onFormSubmit(form.valid)">
```

Comment:
I'm wondering if it is a new approach of using a FromGroup? or it's just your way to work with forms?

Reply:
Hmm.. it's just a regular template-driven Angular forms